Engine/ac/movelist.cpp

```diff
@@ -19,13 +19,13 @@
 using namespace AGS::Common;
 using namespace AGS::Engine;
 
-HSaveError MoveList::ReadFromFile(Stream *in, int32_t cmp_ver)
+HSaveError MoveList::ReadFromFile(Stream *in)
 {
-    if (cmp_ver < 1)
+    int version = in->ReadInt32();
```

Comment:
Um, this contradicts to every other save format, where the format version is only mentioned as "component version" once; here it will be mentioned per each MoveList object. What was the reason to do so?